agent/src/agent/cli/prompt/pipeline/influx.py

```diff
@@ -16,15 +16,25 @@ def prompt_config(self):
         self.prompt_tags()
         self.set_delay()
         self.set_filtering()
+        self.set_transform()
         self.set_uses_schema()
         self.config['timestamp'] = {
             'type': 'unix_ms',
             'name': 'time',
         }
 
+    def set_measurement_name(self):
+        if self.pipeline.query:
+            self.set_query()
```

Comment:
this looks weird, calling set_query in set_measurement_name. Maybe we should just delete the prompt, we don't need it anyway